tomopy/prep/alignment.py

```diff
@@ -49,17 +49,25 @@
 import numpy as np
 import logging
 import warnings
-from skimage import transform as tf
-from skimage.feature import register_translation
+import os
+
+from skimage                import transform              as tf
+from skimage.feature        import register_translation
 from tomopy.recon.algorithm import recon
-from tomopy.sim.project import project
+from tomopy.sim.project     import project
+from tomopy.misc.npmath     import gauss, calc_affine_transform, calc_cummulative_dist
+from scipy.signal           import medfilt, medfilt2d

```

Comment:
Isn't medfilt2d a subset of medfilt? So, why not just use medfilt?

Reply:
medfilt2d is more suitable for processing image